platform/commonUI/general/src/directives/MCTSplitPane.js

```diff
@@ -162,28 +166,27 @@ define(
                 // Getter-setter for the pixel offset of the splitter,
                 // relative to the current edge.
                 function getSetPosition(value) {
-                    var prior = position;
                     if (typeof value === 'number') {
                         position = value;
                         enforceExtrema();
                         updateElementPositions();
 
                         // Pass change up so this state can be shared
-                        if (positionParsed.assign && position !== prior) {
+                        if (positionParsed.assign) {
```

Comment:
Was there a need to remove this check? It was introduced in 3a3829682cc44b4ac93ce13d650d46acd5033aac to limit the number of digest cycles triggered by the split pane.

Reply:
makes sense why it should have been there. I added it back, but removed unused variables min,max